### PR TITLE
SAK-45389 Added markdown docs for the file conversion service

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-45389

### CODE REVIEW

File: kernel/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Contains the implementations to the kernel's API and their associated spring wir
 * Email [implementation](kernel-impl/src/main/java/org/sakaiproject/email/impl) | [components.xml](kernel-impl/src/main/webapp/WEB-INF/email-components.xml)
 * Entity [implementation](kernel-impl/src/main/java/org/sakaiproject/entity/impl) | [components.xml](kernel-impl/src/main/webapp/WEB-INF/entity-components.xml)
 * Event [implementation](kernel-impl/src/main/java/org/sakaiproject/event/impl) | [components.xml](kernel-impl/src/main/webapp/WEB-INF/event-components.xml)
+* FileConversionService [implementation](kernel-impl/src/main/java/org/sakaiproject/content/impl/FILE_CONVERSION_SERVICE.MD) | [components.xml](kernel-impl/src/main/webapp/WEB-INF/content-components.xml)
```
Comment (ern) [inline kernel/README.md:24]: There is already a content impl s few lines up should just use that link

**Review by ern — CHANGES_REQUESTED** [review]

File: kernel/README.md (hunk 1 above)
Comment (adrianfish) [reply kernel/README.md:24]: I don't understand. Do you mean just remove that components link from the file conversion service line, or move the actual file conversion MD file contents into the overall content one?

Comment (ern) [reply kernel/README.md]: yeah just use link to the content components above and put details about the FileConverionService in that README.MD

Comment (adrianfish) [reply kernel/README.md]: It's a pretty big readme. Are you sure you want it in that one file?

Comment (ern) [reply kernel/README.md]: yes

**Comment by adrianfish** [comment]
@ern Done. Created the readme in content/impl

**Review by ern — APPROVED** [review]

**Pull request merged**